comp/otelcol/collector/impl-pipeline/pipeline.go

```diff
@@ -112,6 +123,166 @@ func (c *collectorImpl) stop(context.Context) error {
 	return nil
 }
 
+func (c *collectorImpl) fillFlare(fb flaretypes.FlareBuilder) error {
+	if !c.config.GetBool("otel.enabled") {
+		return nil
+	}
+
+	// request config from Otel-Agent
+	responseBytes, err := c.requestOtelConfigInfo(c.config.GetInt("otel.extension_url"))
+	if err != nil {
+		fb.AddFile("otel/otel-agent.log", []byte(fmt.Sprintf("did not get otel-agent configuration: %v", err))) //nolint:errcheck
```

Comment:
`errcheck` should ignore the flarebuilders method (see [this](https://github.com/DataDog/datadog-agent/blob/main/.golangci.yml#L102)).

Reply:
Ah I see. I had, in an earlier PR, split flare/types into a non-fx (compdef) version, which this linter doesn't cover. I'm now adding a rule to acknowledge these errchecks as well.